pkg/packagekit/updates.jsx

```diff
@@ -172,9 +172,9 @@ function HeaderBar(props) {
             actionButton = <button className="btn btn-default" onClick={props.onRefresh} >{_("Check for Updates")}</button>;
         if (props.timeSinceRefresh !== null) {
             lastChecked = (
-                <span style={ {paddingRight: "3ex"} }>
+                <React.Fragment>
                     { cockpit.format(_("Last checked: $0 ago"), moment.duration(props.timeSinceRefresh * 1000).humanize()) }
-                </span>
+                </React.Fragment>
```

Comment:
This is just a single text element, so I don't think it needs to be a fragment at all. The span was just done for the paddingRight, but that's now gone anyway (and probably never worked because of unsafe-inline CSP)